pkg/kube/client.go

```diff
@@ -232,7 +231,7 @@ type client struct {
 }
 
 // newClientInternal creates a Kubernetes client from the given factory.
-func newClientInternal(clientFactory util.Factory, revision string) (*client, error) {
+func newClientInternal(clientFactory util.Factory, namespace, revision string) (*client, error) {
```

Comment:
shouldn't this be a list of namespaces?

Reply:
@bison is working on getting namespace list functionality. For now we need this single namespace to avoid a regression from 1.6 -> 1.7